parsec-3.0/help.txt

```diff
@@ -0,0 +1,55 @@
+You need to give arguments to execute installed benchmarks:
+		docker run [repository/tag] [ACTION] [PACKAGE] [BUILDCONF] [INPUT] [-n THREADS]
+
+All input workloads are compressed, to know how to uncompress one, type :
+		docker run [repository/tag] uncompress-help   
```

Comment:
It is, I just thought it was a good idea to split the output so that it's not too long